### ISSUE TITLE
Investigate using CodeMirror screenReaderLabel

### ISSUE THREAD

**Issue by MasterOdin** [Submitty/Submitty#5296]
The newest version of CodeMirror (5.53) added a new option [screenReaderLabel](https://codemirror.net/doc/manual.html#option_screenReaderLabel) which can be applied to CodeMirror text areas to help with accessibility. We should investigate this and add them as appropriate around the site if they do prove useful.

**Comment by elihschiff** [comment]
I tested what this does with a screen reader. Using the picture below as an example. When you use the screen reader, it will first read the text above the textbox. Then when you tab to the textbox it will then read off the string you pass in for `screenReaderLabel` if you dont pass in a string it will just read out the word "textarea". If you are using a screen reader but you dont press tab and instead navigate with the mouse, I was unable to get it to make a difference. 

I am not sure adding a string for `screenReaderLabel` really does much here as the text before a textarea should always be enough to know what it is for, and so being let know you are selected on the following textarea by being told "textarea" is probably more helpful than any custom text we could provide instead.

If you think that there is still helpful text we could add though `screenReaderLabel`, I am happy to add it, but I don't think we really would benefit.

![image](https://user-images.githubusercontent.com/18558130/83212755-5b446d00-a12e-11ea-8870-e54abe4fc476.png)
